src/router/routes.js

```diff
@@ -118,8 +118,8 @@ module.exports = [
 		}
 	},
 	{
-		path: '/checkout/eco-challenge/thanks',
-		component: () => import('@/pages/Thanks/ThanksPageEcoChallenge'),
+		path: '/checkout/iwd-challenge/thanks',
```

Comment:
Do we need to redirect the old path or is it not linked to from anywhere?

Reply:
Its not linked.